app/helpers/application_helper.rb

```diff
@@ -116,9 +119,36 @@ def smart_annotation_parser(text)
     new_text = new_text.gsub(sa_user) do |el|
       match = el.match(sa_user)
       user = User.find_by_id(match[2].base62_decode)
-      if user
-        "<span>#{image_tag avatar_path(user, :icon_small)} " \
-        "#{user.full_name}</span>"
+      organization ||= current_organization
+      if user && organization
+        user_org = user
+                   .user_organizations
+                   .where('user_organizations.organization_id = ?',
+                          organization).first
+        user_description = '<div class=\'pull-left\'>'
```

Comment:
Just a styling suggestion, up to you - wouldn't it make sense to use `%Q{ ... }`, or `<<EOF ... EOF` here since it's a big block of escaped text?